api/rights.proto

```diff
@@ -66,6 +66,8 @@ enum Right {
   RIGHT_APPLICATION_SETTINGS_API_KEYS = 17;
   // The right to view and edit application collaborators.
   RIGHT_APPLICATION_SETTINGS_COLLABORATORS = 18;
+  // The right to view and edit application packages and associations.
+  RIGHT_APPLICATION_SETTINGS_PACKAGES = 56;
```

Comment:
@kschiffer @bafonins what is needed to make this manageable in the Console?

Reply:
We need a respective message for this in `messages.json` as we have for the other rights. Then it should be manageable in the console right away.

Reply:
See `pkg/ttnpb/i18n.go`